Gopkg.lock

```diff
@@ -2,23 +2,26 @@
 
 
 [[projects]]
-  digest = "1:218ae0d3f01ab221e2a62493a88e72189fb4941b22f60984277c22f664d34c46"
```

Comment:
Your PR contains only the updated scripts in `test-infra`, but you updated several digests/revisions from other vendored dirs. Please restrict the changes to `test-infra` dir only.

Reply:
updated